modules/cli/pages/index.adoc

```diff
@@ -174,8 +174,12 @@ where `<link-name>` is the name of the link specified during creation.
 When linking sites, you can assign a cost to each link to influence the traffic flow. By default, link cost is set to `1` for a new link.
 In a {service-network}, the routing algorithm attempts to use the path with the lowest total cost from caller to target service.
 
-If you have more than two sites and services distributed across different clusters, you might want a calling service to favor a particular target or link.
-In this case, you can specify a cost of greater than `1`.
+If you have services distributed across different sites, you might want a calling service to favor a particular target or link.
+In this case, you can specify a cost of greater than `1` on the alternative links to reduce the usage of those links.
+For TCP, the exact usage level of a link is determined by the number of open connections.
+When the number of open connections exceeds the cost, the next request is likely to open a connection to a different target.
```

Comment:
Cost acts as a threshold for using a particular path/link in the network.  So, if the number of connections is _less_ than the cost of a link, that link will not be used.

Think of a bucket with holes in its side walls.  The holes are outgoing links to destinations or remote routers with destinations.  The water poured into the bucket is the offered load (i.e. the client requests).  The higher cost links are drilled higher in the bucket walls.  If the pour rate is slow enough, the lower holes will drain all the water.  If the rate increases and the water level increases, the higher cost holes will start draining water.

Reply:
This analogy is slightly flawed because the bucket will never hold standing water (i.e. the water level is below the lowest hole).  I guess we could say the bucket has a feature where its floor automatically rises to the level of the lowest hole.

So, if we are running a steady flow of traffic through a link with cost=10 and that link becomes unusable because its downstream server disconnects, another link with cost=1000000 will immediately pick up all the traffic since it has become the lowest cost link.

Reply:
This looks good.

There's one aspect we didn't discuss which might be important.  This cost algorithm operates from the perspective of the client's location.  If I have two cloud locations with a relatively high cost between them and services available on both sides, then clients on both sides will be served preferentially by their local service.  Only if a service fails or the load on a side crosses the threshold will there be any cross-site communication.

Reply:
made a few changes to push 'client' and 'server' (now that those words are in console). Also kinda snuck  your point into a new 'observe' step, help?